### PR TITLE
💄 Add Header and Footer

### LINKED ISSUE #3: Add Design Components
- [x] Add Fonts
- [x] Style h1, h2, h3, h4, a, div
- [x] Add responsive Buttons with label and icon
- [x] Style Header
- [x] Style Footer

### PR DESCRIPTION
**Pull request by m-wiemers**
Fix #3 
- style Footer
- style Header
- add header and footer to index

### CODE REVIEW

File: components/DateInput/DateInput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export default {
+  title: "Common/DateInput",
+  component: DateInput,
+};
+
+const Template: Story<DateProps> = (args) => <DateInput {...args} />;
+
+export const StartDate = Template.bind({});
+StartDate.args = {
+  label: "When does it start?",
```
Comment (RonjaMeinecke) [inline components/DateInput/DateInput.stories.tsx]:
```suggestion
  label: "trip start",
```

File: components/DateInput/DateInput.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+};
+
+export const EndDate = Template.bind({});
+EndDate.args = {
+  label: "When does it end?",
```
Comment (RonjaMeinecke) [inline components/DateInput/DateInput.stories.tsx]:
```suggestion
  label: "trip end",
```

**Review by DNaunin — APPROVED** [review]
LGTM! Beautiful formatting of your svg-files! 💄

File: components/citieSelect/SelectCity.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export default {
+  title: "Common/SelectCity",
+  component: SelectCity,
+};
+
+const Template: Story<SelectProps> = (args) => <SelectCity {...args} />;
+
+export const StartCity = Template.bind({});
+StartCity.args = {
+  label: "Where does it start?",
```
Comment (RonjaMeinecke) [inline components/citieSelect/SelectCity.stories.tsx]:
```suggestion
  label: "trip start",
```

File: components/citieSelect/SelectCity.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  value: ["Brisbane", "Sydney", "Cairns", "Byron"],
+};
+
+export const EndCity = Template.bind({});
+EndCity.args = {
+  label: "Where does it end?",
```
Comment (RonjaMeinecke) [inline components/citieSelect/SelectCity.stories.tsx]:
```suggestion
  label: "trip end",
```

**Review by RonjaMeinecke — APPROVED** [review]
great progress 🦘

**Pull request merged**